docs/content/preview/explore/ysql-language-features/going-beyond-sql/tablespaces.md

```diff
@@ -14,34 +14,42 @@ aliases:
+The ability to control the placement of tables in a fine-grained manner provides the following advantages:
+
+- Tables with critical information can have higher replication factor and increased fault tolerance compared to the rest of the data.
+- Based on the access pattern, a table can be constrained to the region or zone where it's accessed most frequently.
+- A table can have an index with an entirely different placement policy, thereby boosting the read performance without affecting the placement policy of the table itself.
+- Coupled with table partitioning, tablespaces can be used to implement row-level geo-partitioning. This allows pinning the rows of a table in different geo-locations based on the values of certain columns in that row.
+
+## Data placement and latency
+
+In a distributed cloud-native database such as YugabyteDB, the location of tables and indexes plays an important role in achieving optimal performance for any workload. The following diagram illustrates the ping latencies amongst nodes in a geo-distributed cluster. Nodes closer to each other communicate with visibly lesser latency than nodes physically far away from each other.
 
 ![Cluster Ping Latencies](/images/explore/tablespaces/cluster_ping_latencies.png)
 
-Given the impact of distance on node-to-node communication, it is highly useful to be able to specify at a table level, how its data should be spread across the cluster. This way, you can move tables closer to their clients and decide which tables actually need to be geo-distributed. This can be achieved using YSQL Tablespaces. YSQL Tablespaces are entities that can specify the number of replicas for a set of tables or indexes, and how each of these replicas should be distributed across a set of cloud, regions, zones.
+Given the impact of distance on node-to-node communication, it's beneficial to be able to specify at a table level, how its data should be spread across the cluster. This way, you can move tables closer to their clients and decide which tables actually need to be geo-distributed. You do this using YSQL tablespaces. Using tablespaces you can specify the number of replicas for a set of tables or indexes, and how each of these replicas should be distributed across a set of cloud, regions, and zones.
 
-This document describes how to create the following:
-* A cluster that is spread across multiple regions across the world.
-* Tablespaces that specify single-zone, multi-zone and multi-region placement policies.
-* Tables associated with the created tablespaces.
+## Cluster setup
 
-This can be summarized in the following diagram:
-![Overview Cluster Diagram](/images/explore/tablespaces/overview_cluster_diagram.png)
+The following example describes how to create the following:
```

Comment:
The following example describes how to create the following:  -->  Too many "following" in one short sentence